docker/setup_configuration/data.yaml

```
@@ -0,0 +1,8 @@
+oidc_db_config_enable: true
+oidc_db_config_admin_auth:
+  oidc_rp_client_id: client-id
```

The new configuration with `items` and `identifier` is missing here:
```suggestion
    items:
    - identifier: admin-oidc
      oidc_rp_client_id: client-id
      oidc_rp_client_secret: secret
      endpoint_config:
```